README.MD

```diff
@@ -0,0 +1,62 @@
+[![Build Status](https://travis-ci.org/stryker-mutator/stryker-jest-runner.svg?branch=master)](https://travis-ci.org/stryker-mutator/stryker-jest-runner)
+[![Gitter](https://badges.gitter.im/stryker-mutator/stryker.svg)](https://gitter.im/stryker-mutator/stryker?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+![Stryker](https://github.com/stryker-mutator/stryker/raw/master/stryker-80x80.png)
+
+# Stryker-jest-runner
+
+## Installation
+Install stryker-jest-runner locally within your project folder, like so:
+
+```bash
+npm i --save-dev stryker-jest-runner
+```
+
+## Peer dependencies
+The stryker-jest-runner is a plugin for Stryker to enable Jest as a test runner. As such, you should make sure you have the correct versions of its dependencies installed:
+
+- jest
+- stryker-api
+
+For the minimum supported versions, see the peerDependencies section in package.json.
+
+## Configuration
+Set your test runner to `jest` in the stryker config, optionally provide the project you're working on for a preset configuration.
+We currently only support `react` preset, enable it by adding the following config entry to the stryker config:
+
+```javascript
```

Comment:
Could you add two bits of config? One for react projects (which also sets the testRunner property) and one for any other project?

Reply:
You specify it later on, but I would expect the testRunner property here as well.